### ISSUE TITLE
Action should not fail if optional files do not exist

### ISSUE THREAD

**Issue by milis92** [r0adkll/upload-google-play#161]
**Describe the bug**

TLDR; Action should not fail if the **optional** files are not present on the disc, even if they are passed to the action.  

1. They are optional; they shouldn't prevent the action from executing its primary use case.

2. Since GitHub actions have no support for IF-THEN statements, it's sometimes hard to add the files only if they exist on the disc without overloading the action with multiple combinations.   
For example, multi-flavor builds where not all flavors publish the mapping file, or if we want to create a deployment [template workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows). 

For the second, let's observe the following action configuration:

#### Where we have a job template
```yml

on:
  workflow_call:
    inputs:
      BUILD_ENVIRONMENT:
        required: false
        default: staging
        description: 'Project environment flavor to be deployed (production or staging)'
        type: string
      BUILD_TYPE:
        required: false
        default: debug
        description: 'Android build type to be deployed (debug or release)'
        type: string
      DEPLOYMENT_TRACK:
        required: false
        default: internalsharing
        description: 'Play store deployment track to be used (internalsharing, alpha, beta, production)'
        type: string

jobs:
  deploy:
    name: Deploy
    runs-on: ubuntu-latest
    
     # ... Action setup steps 
    
     # Construct Android Flavor 
      - name: Construct Android Flavor
        env:
          BUILD_TYPE: ${{ inputs.BUILD_TYPE }}
          BUILD_ENV: ${{ inputs.BUILD_ENVIRONMENT }}
        run: |
          echo "ANDROID_VARIANT=$BUILD_ENV${BUILD_TYPE^}" >>${GITHUB_ENV}

      # Create the release bundle
      - name: Build the android bundle
        id: bundle
        run: ./gradlew bundle$ANDROID_VARIANT

      - name: Deploy
        uses: r0adkll/upload-google-play@v1.0.19
        id: upload
        with:
          serviceAccountJsonPlainText: ${{ secrets.PLAYSTORE_SERVICE_ACCOUNT }}
          packageName: com.example
          releaseFiles: app/build/outputs/bundle/${{ env.ANDROID_VARIANT }}/*.aab
          track: ${{ inputs.DEPLOYMENT_TRACK }}
          status: inProgress
          inAppUpdatePriority: 2
          userFraction: 0.2
```

#### And a template consumer that's publishing stagingDebug flavor to the internalsharing
```yml
jobs:
  deploy:
    name: Deploy
    uses: ./.github/workflows/template_deploy.yml
    with:
      BUILD_ENVIRONMENT: 'staging'
      BUILD_TYPE: 'debug'
      DEPLOYMENT_TRACK: 'internalsharing'
    secrets: inherit
```

#### And a template consumer that's publishing productionRelease flavor to the production track
```yml
jobs:
  deploy:
    name: Deploy
    uses: ./.github/workflows/template_deploy.yml
    with:
      BUILD_ENVIRONMENT: 'production'
      BUILD_TYPE: 'release'
      DEPLOYMENT_TRACK: 'production'
    secrets: inherit
```
---
With the current implementation of the action, there is no reasonably clean way to check if the flavor will generate mapping files or native debug symbols. So we can either not pass the optional files or overload the action configuration with all the variations of supporting and checking optional files.

--- 
### Desired behavior;
Action should display a warning if any optional files are missing; treat them as `undefined` and continue with the execution.

**Comment by milis92** [comment]
Created a [PR ](https://github.com/r0adkll/upload-google-play/issues/161) that fixes this

**Comment by boswelja** [comment]
PR merged, thanks for your contribution!
